### ISSUE TITLE
2.2 : docker doesn't like ":" in name of project

### ISSUE THREAD

**Issue by Raizo62** [GNS3/gns3-gui#3437]
Hi

I create a topolgy with the name "lab:name" (so, with a ":" in, the name).

When i put on the lab a docker container, GNS3 displays this error message : 

`2023-02-12 09:40:31 ERROR template_manager:226 Error while creating node from template: Docker has returned an error: 500 invalid mode: /gns3volumes/etc/network`

**Comment by grossmj** [comment]
Do you have the issue with any Docker container?

**Comment by Raizo62** [comment]
Yes : now, i have tried with an alpine container (but with my patches on GNS3).
When i opened this issue, i had removed my patches to verify.

You can't reproduce the issue ?

**Comment by Raizo62** [comment]
@b-ehlers : thanks :-) 

> I suggest to add this to 3.0 instead, to keep 2.2 stable. I suggest to not change 2.2 at this late stage, I would mark this bug as a known bug for 2.2. But, of course, Jeremy has to decide.

While waiting for the patch in version 3, we can forbid the character ":" in the name of projects of version 2.2
